src/Behat/Behat/Definition/Attribute/When.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Behat.
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Behat\Behat\Definition\Attribute;
+
+/**
+ * Represents an Attribute for When steps
+ */
+#[\Attribute(\Attribute::TARGET_METHOD)]
```

Comment:
it must be allowed to be repeated as the same method can be registered multiple times with different patterns

Reply:
They are now repeatable (which is also verified through `features/attributes.feature`) 👍